mkdocs.yml

```diff
@@ -64,6 +64,8 @@ markdown_extensions:
   - tables
 
 plugins:
+  - mkdocs-typer2:
```

Comment:
You said documentation generation for the CLI was a semi-manual process. You should probably document it.

Reply:
good point, and I need to remove this plugin